basicsr/metrics_Lotayou/__init__.py

```diff
@@ -0,0 +1,23 @@
+from copy import deepcopy
+
```

Comment:
I think the original `metrics` could also support fid~ 
If not, please open a discussion.

Reply:
Will leave this in till next PR~